### DISCUSSION TITLE
Is there any nn module or layer is suitable for edge classification in torch_geometric?

### DISCUSSION THREAD

**Discussion by ANYMS-A** [pyg-team/pytorch_geometric#9843] [Q&A]
Hi, I would like to know is there any tutorial or example which deal with the edge classification in the graph? Thank!

**Comment by Romain-Nicolle** [comment]
Hello,

I am not an expert but I think you are looking for something like [this](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/link_pred.py) ? There are a lot of examples of pyg code [there](https://github.com/pyg-team/pytorch_geometric/tree/master/examples) in node/link/graph classification. 

You also have several tutorials [there](https://pytorch-geometric.readthedocs.io/en/latest/get_started/colabs.html), including one on [link prediction](https://colab.research.google.com/drive/1xpzn1Nvai1ygd_P5Yambc_oe4VBPK_ZT?usp=sharing) and even one on [link regression](https://colab.research.google.com/drive/1N3LvAO0AXV4kBPbTMX866OwJM9YS6Ji2?usp=sharing).

Hope that helps.

Best,
